### DISCUSSION TITLE
blog/new-module-psmqtt/

### DISCUSSION THREAD

**Discussion by giscus** [hanpq/getps.dev#16] [Comments]
# blog/new-module-psmqtt/

Powershell module to work with MQTT

https://getps.dev/blog/new-module-psmqtt/

**Comment by stefanstranger** [comment]
Thanks for creating this PowerShell module! I used it to send MQTT messages to my Ulanzi Pixel Clock. 

See this twitter message: https://x.com/sstranger/status/1740385654318760267?s=20
